.github/workflows/ci.yml

```diff
@@ -10,18 +10,35 @@ jobs:
         python-version:
           - "3.10"
           - "3.11"
+        pydantic-version:
+          - "^1.10"
+          - "^2.0"
```

Comment:
for the `make lint` it's useless

Reply:
the job is called "lint" but we have a "mypy" checker on it... should we really remove it ?